scenarios/kubernetes_kops_aws.py

```diff
@@ -200,7 +200,8 @@ def main(args):
 
     # Assume we're upping, testing, and downing a cluster by default
     PARSER.add_argument(
-        '--cluster', default='e2e-kops-aws-canary.test-aws.k8s.io',
+        '--cluster',
+        default=os.environ.get('KOPS_E2E_CLUSTER_NAME'),
```

Comment:
you want to pass it down to --kops-cluster right?

`os.environ.get('KOPS_E2E_CLUSTER_NAME')` will be None